sycl/unittests/pi/cuda/test_interop_get_native.cpp

```diff
@@ -97,3 +84,7 @@ TEST_F(CudaInteropGetNativeTests, interopTaskGetBufferMem) {
     });
   });
 }
+
+INSTANTIATE_TEST_CASE_P(
+    OnCudaPlatform, CudaInteropGetNativeTests,
+    ::testing::ValuesIn(pi::getPlatformsWithVersion("CUDA")), );
```

Comment:
Just curious, what does this comma in the end does?

Reply:
Since `INSTANTIATE_TEST_CASE_P` is a variadic macro, so after the 3rd argument it may take 0 or more arguments. However, though having only the 3 required arguments is totally valid for this macro, it can cause a warning which can in turn be seen as an error. This is the easiest way to prevent this warning.